src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1469,12 +1470,28 @@ def sub_msg_handler(self, msg_item):
             except Exception as e:
                 syslog.syslog(syslog.LOG_ERR, '[bgp cfgd] Failed handling config DB update with exception:' + str(e))
                 logging.exception(e)
+
+    def listen_thread(self, timeout):
+        self.sub_thread_running = True
+        sub_key_space = "__keyspace@{}__:*".format(self.get_dbid(self.db_name))
+        self.pubsub.psubscribe(sub_key_space)
+        while self.sub_thread_running:
+            msg = self.pubsub.get_message(timeout, True)
+            if msg:
+                self.sub_msg_handler(msg)
+
+        self.pubsub.punsubscribe(sub_key_space)
```

Comment:
>punsubscribe [](http://example.com/codeflow?start=20&length=12)

Found similar code in [https://github.com/sonic-net/sonic-swss-common/blob/85f3776ae6da44a083c617ff4ce6e54e1e2d5dac/common/configdb.h#L78](https://github.com/sonic-net/sonic-swss-common/blob/85f3776ae6da44a083c617ff4ce6e54e1e2d5dac/common/configdb.h#L78). Do we also need `punsubscribe` there?

Reply:
Yes, I think we also unsubscribe after exit the while loop in ConfigDBConnector, I will create PR for this.